src/logic/schedule-parser/children-info.parser.ts

```diff
@@ -51,7 +52,7 @@ export class ChildrenInfoParser implements ChildrenInfoProvider {
 
     if (slicedChildrenRow.length !== this.metaData.dayCount) {
       this.logLoadFileError(
-        "Sekcja dzieci nie ma oczekiwanych wymiarów. Dla brakujących dni przyjęto, że liczba dzieci wynosi 0"
+        "Sekcja dzieci nie ma oczekiwanych wymiarów. Dla brakujących dni przyjęto, że liczba dzieci wynosi 0."
```

Comment:
Brakuje mi trochę bolda w przypadku błędów z parsera, co jest trochę niekonsekwentne w stosunku do tego co mamy w drawerze z błędami. Dodatkowo poprawia czytelność, więc fajnie jakbyś tutaj też to wprowadziła.

Reply:
@Ehevi